docs/src/main/asciidoc/duplicated-context.adoc

```diff
@@ -76,14 +76,14 @@ In Quarkus, when you execute reactive code, you run in a _Context_, representing
 @Path("/hello1")
 public String hello1() {
    Context context = Vertx.currentContext();
-   return "Hello, you are running on context: %s".formatted(context);  <1>
+   return "Hello, you are running on context: %s".formatted(context, Thread.currentThread());  <1>
```

Comment:
don't you need a second %s?

Reply:
Ah, yes the whole `and on thread %s` is missing here...